### PR TITLE
Update Tournaments index page

### PR DESCRIPTION
**Pull request by fajar13k**
- Add Intermediate Global Taiko Showdown to the index page
- Update Catch the Beat Indonesia Cup timeline (add previous iterations)
- Update Catch French Cup timeline (add previous iterations)
- Update CJB osu!mania 4K Tournament table format (no beautify)

---

### CODE REVIEW

File: wiki/Tournaments/en.md [hunk 1]
```diff
@@ -141,6 +153,11 @@ Unofficial tournaments/competitions hosted by the communities.
 | --- | --- | --- | --- | --- | --- |
 | [The Heptakey osu!mania Cup #1](/wiki/Tournaments/THOM_CUP/1) | 2018-04-16 | 2018-07-15 | ![][flag_ES] [aitor98](https://osu.ppy.sh/users/3154852) | ![][flag_IT] [Yomiel](https://osu.ppy.sh/users/3461860) | ![][flag_CL] [Skalim](https://osu.ppy.sh/users/2225008) |
 
+### ![](/wiki/shared/mode/mania.png) CJB osu!mania 4K Tournament
```
Comment (TPGPL) [inline wiki/Tournaments/en.md]: Add a break line after the header

File: wiki/Tournaments/fr.md [hunk 2]
```diff
@@ -109,16 +109,28 @@ Tournois/Compétitions non officiels organisés par la communauté.
 | Taiko: Last Man Standing 4 | 2017-09-23 | 2017-10-28 | Tarako Spa | Can We Go Back | easy to giveup |
 | [Taiko Last Man Standing: Shift Cup](/wiki/Tournaments/LMS/Shift/1) | 2018-06-09 | 2018-07-15 | V E R Y S T R O N G T E A M | あ | Ｆａｓｔ ＆ Ｃｈｉｌｌ 早く |
 
+### ![](/wiki/shared/mode/taiko.png) Intermediate Global Taiko Showdown
+
+| Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
```
Comment (TPGPL) [inline wiki/Tournaments/fr.md]: not translated header, nor country names

File: wiki/Tournaments/id.md [hunk 3]
```diff
@@ -109,16 +109,28 @@ Turnamen/Kompetisi tidak resmi yang diselenggarakan oleh komunitas.
 | Taiko: Last Man Standing 4 | 2017-09-23 | 2017-10-28 | Tarako Spa | Can We Go Back | easy to giveup |
 | [Taiko Last Man Standing: Shift Cup](/wiki/Tournaments/LMS/Shift/1) | 2018-06-09 | 2018-07-15 | V E R Y S T R O N G T E A M | あ | Ｆａｓｔ ＆ Ｃｈｉｌｌ 早く |
 
+### ![](/wiki/shared/mode/taiko.png) Intermediate Global Taiko Showdown
+
+| Nama | Mulai | Selesai | ![Mahkota Emas](/wiki/shared/GCrown.png "Juara satu") | ![Mahkota Perak](/wiki/shared/SCrown.png "Juara dua") | ![Mahkota Perunggu](/wiki/shared/BCrown.png "Juara tiga") |
+| --- | --- | --- | --- | --- | --- |
+| [Intermediate Global Taiko Showdown 2018](/wiki/Tournaments/IGTS/2018) | 2018-08-04 | 2018-09-16 | ![][flag_MY] Malaysia | ![][flag_NL] Netherlands | ![][flag_US] United States |
```
Comment (TPGPL) [inline wiki/Tournaments/id.md]: not translated country names

**Comment by Joehuu** [comment]
Change `USA` instances to `United States`.

**Comment by Joehuu** [comment]
I already added the link on the recently merged TWC#3 PR on all updated languages.

Idk why it's not conflicting (it shows it as a diff here).

**Pull request merged**